build-system/tasks/extension-helpers.js

```diff
@@ -509,6 +515,32 @@ function buildExtensionCss(path, name, version, options) {
   return Promise.all(promises);
 }
 
+async function buildNpm(path, name, version, options) {
+  const {binaries, external} = options.npm;
+  const promises = binaries.map((filename) => {
+    const {name} = pathParse(filename);
+    const esm = argv.esm || argv.sxg || false;
+    return esbuildJs(
+      path + '/',
+      filename,
+      `${path}/build`,
+      Object.assign(options, {
+        toName: `${name}${options.minify ? '' : '.max'}.${esm ? 'm' : ''}js`,
+        latestName: '',
+        outputFormat: esm ? 'esm' : 'cjs',
+        wrapper: '',
+        external,
+      })
+    );
+
+    // If an incremental watch build fails, simply return.
```

Comment:
Looks like an unreachable code?

Reply:
Yah, this was copy pasta from `buildExtensionJs` path. Fixed.